tests/components/weatherflow_cloud/test_config_flow.py

```diff
@@ -0,0 +1,44 @@
+"""Test the WeatherflowCloud config flow."""
+
+from homeassistant import config_entries
+from homeassistant.components.weatherflow_cloud.const import DOMAIN
+from homeassistant.const import CONF_API_TOKEN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+
+async def test_config_2(hass: HomeAssistant, mock_get_stations_combined) -> None:
```

Comment:
Can we have a bit separate tests so
- Works fine
- Side-effects (better if you parameterize it) which then recover and ends with creating the entry.
- abort because already configured

Reply:
Done! :)